src/logging/consoleOverride.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable prefer-spread */
+/* eslint-disable prefer-rest-params */
+
+import { format } from 'util';
+import { Logger } from 'winston';
+
+/**
+ * Override console methods with a Winston.Logger.
+ *
+ * @param logger
+ */
+export function consoleOverride(logger: Logger): void {
+	console.log = function (...args: unknown[]) {
+		// Sacrereligious typecasting explained:
+		//
+		// `args as [string]`: fomat @types dictate that it needs an array of at least length 1. However,
+		// from testing this is not neccesary, so we override the type as a string tuple.
+		//
+		// `(format.apply(format, args as [string]) as unknown) as object: TS incorrectly says the we
+		// need a object as an argument to `info.call`. However, it will except a string perfectly fine,
+		// which is what `format.apply` returns.
+		logger.info.call(
+			logger,
+			(format.apply(format, args as [string]) as unknown) as object
+		);
+	};
+	console.info = function (...args: unknown[]) {
```

Comment:
This function has a lot of duplication.
Basically, what you are doing is to say: "for function A, B and C, please use this implementation".

 You can make it shorter and more concise using:
```
['info', 'error', 'warn', 'log'].map (level => {
     console[level] = function (...);
}
```